src/api/.rubocop.yml

```diff
@@ -153,6 +153,10 @@ Style/ZeroLengthPredicate:
 Style/RescueModifier:
   Enabled: true
 
+# Checks for methods invoked via the . operator (like FileUtils::rmdir instead of FileUtils.rmdir).
```

Comment:
This comment is wrong, it should be something like:

_This cop checks for methods invoked via the **::** operator instead of the . operator_

I think that the example _like FileUtils::rmdir instead of FileUtils.rmdir_ is not needed here.